services/frontend/src/scenes/ProjectDetails/ProjectDetailsLayout.js

```diff
@@ -75,16 +76,38 @@ const ProjectContent = ({...sectionProps}) => (
 );
 
 const ProjectInfo = ({project}) => {
+    const [projectInfoDialogOpen, setProjectInfoDialogOpen] = useState(false);
+
+    const openDialog = () => {
+        setProjectInfoDialogOpen(true);
+    };
+
+    const closeDialog = () => {
+        setProjectInfoDialogOpen(false);
+    };
+
     return (
-        <Paper className="project-details-card">
-            <div className="project-info-card-active-section">
-                <ActiveBadge isActive={Project.isActive(project)} />
-            </div>
-            <div className="project-info-card-content-section">
-                <h3 className="project-info-card-name">{project.name}</h3>
-                <p className="project-info-card-description">{project.description}</p>
-            </div>
-        </Paper>
+        <>
+            <ProjectInfoDialog
+                closeDialog={closeDialog}
+                open={projectInfoDialogOpen}
+                project={project}
```

Comment:
same issue as last time

Reply:
This will be fixed in CDHSH-99 when I connect dialog to the backend.